next.config.js

```diff
@@ -30,15 +30,31 @@ const nextConfig = {
       ],
       ["/mirror/rockset", "/mirror/sinks/rockset"],
       ["/mirror/timescale", "/mirror/sinks/timescale"],
-      ["/mirror/webhooks", "/indexing/webhooks"],
-      ["/mirror/sinks/webhooks", "/indexing/webhooks"],
+      ["/mirror/webhooks", "/subgraphs/webhooks"],
+      ["/mirror/sinks/webhooks", "/subgraphs/webhooks"],
       [
         "/references/instant-subgraphs-config",
         "/references/instant-subgraphs-configuration",
       ],
       ["/streams", "/fusion"],
       ["/streams/cross-chain-subgraphs", "/fusion/cross-chain-subgraphs"],
       ["/streams/transforms", "/fusion/transforms"],
+      ["/indexing", "/subgraphs"],
+      [
+        "/indexing/dedicated-subgraph-indexing",
+        "/subgraphs/dedicated-subgraphs",
+      ],
+      [
+        "/indexing/deploying-subgraphs-base",
+        "/subgraphs/deploying-subgraphs-base",
+      ],
+      ["/indexing/deploying-subgraphs", "/subgraphs/deploying-subgraphs"],
+      ["/indexing/instant-subgraphs", "/subgraphs/instant-subgraphs"],
+      [
+        "/indexing/shared-subgraph-indexing",
+        "/subgraphs/shared-subgraph-indexing",
```

Comment:
Looking at the available files, this should probably change.

```suggestion
        "/subgraphs/shared-subgraphs",
```

Reply:
nice catch